tests/improver-nowcast-extrapolate/07-model_winds.bats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bats
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2018 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+. $IMPROVER_DIR/tests/lib/utils
+
+@test "extrapolate basic" {
+  improver_check_skip_acceptance
+  KGO0="optical-flow/speeds/kgo0.nc"
```

Comment:
I've done a wholesale rearrange at this prompting. The directory structure was different to all the other CLI tests, the directory name not matching the CLI name. I have moved ```optical-flow``` to become ```nowcast-optical-flow``` and I have created ```nowcast-extrapolate```. There is still crossover between the two because they are so closely linked, e.g. the nowcast-optical-flow tests that use the extrapolate option now find their KGO in the nowcast-extrapolate directory. There is no perfect solution to this. We could duplicate the directory structure containing the KGO in both the nowcast-optical-flow and nowcast-extrapolate directories and then use soft links to duplicate the data. Would that be preferable?

Reply:
I've looked through this, and the changes make sense to me. I don't think we need to duplicate the directory structure, I'm happy with what you've done.